### PR TITLE
spadfs-utils

### PR DESCRIPTION
**Pull request by Goldsrc233**

### CODE REVIEW

File: sys-fs/spadfs-utils/spadfs-utils-1.0.18.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+S="${WORKDIR}/spadfs-${PV}"
+
+src_prepare() {
+	default
+}
```
Comment (liangyongxiang) [inline sys-fs/spadfs-utils/spadfs-utils-1.0.18.ebuild]: If only one default is called here, the entire src_prepare function can be deleted.

**Comment by liangyongxiang** [comment]
Multiple git commits can be rebase into one.

File: sys-fs/spadfs-utils/spadfs-utils-1.0.18.ebuild (hunk 1 above)
Comment (Goldsrc233) [reply sys-fs/spadfs-utils/spadfs-utils-1.0.18.ebuild]: ok

**Comment by Goldsrc233** [comment]
> Multiple git commits can be rebase into one.

让我先康康教程，第一次用git不知道怎么rebase

**Pull request merged**